presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergMetadata.java

```diff
@@ -696,4 +731,285 @@ public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTab
                 .map(id -> IcebergUtil.resolveSnapshotId(table, id))
                 .or(() -> Optional.ofNullable(table.currentSnapshot()).map(Snapshot::snapshotId)));
     }
+
+    @Override
+    public void createMaterializedView(ConnectorSession session, SchemaTableName viewName, ConnectorMaterializedViewDefinition definition, boolean replace, boolean ignoreExisting)
+    {
+        HiveIdentity identity = new HiveIdentity(session);
+        Optional<Table> existing = metastore.getTable(identity, viewName.getSchemaName(), viewName.getTableName());
+
+        // It's a create command where the materialized view already exists and 'if not exists' clause is not specified
+        if (!replace && existing.isPresent() && !ignoreExisting) {
```

Comment:
I incorporated first part of the comment.

If the view exists, the storage table must exist since storage table is created when view is created or replaced. If storage table was dropped by the user (using `drop table storage_table`), the refresh operation will fail.